python/hidet/utils/model_translator/tests/classes.py

```diff
@@ -0,0 +1,91 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Better to move the tests `classes.py` and `functions.py` to `tests/utils/model_translator/` folder.